apps/full-site-editing/package.json

```diff
@@ -16,49 +16,49 @@
-		"full-site-editing": "calypso-build --source='dotcom-fse'",
-		"dev:full-site-editing": "npm run full-site-editing",
-		"build:full-site-editing": "NODE_ENV=production npm run full-site-editing",
-		"posts-list-block": "calypso-build --source='posts-list-block'",
-		"dev:posts-list-block": "npm run posts-list-block",
-		"build:posts-list-block": "NODE_ENV=production npm run posts-list-block",
-		"common": "calypso-build --source='common'",
-		"dev:common": "npm run common",
-		"build:common": "NODE_ENV=production npm run common",
-		"blog-posts-block-editor": "calypso-build --source='blog-posts-block/editor'",
-		"blog-posts-block-view": "calypso-build --source='blog-posts-block/view'",
-		"blog-posts-block": "npm-run-all --parallel blog-posts-block-*",
-		"dev:blog-posts-block": "npm run blog-posts-block",
-		"build:blog-posts-block": "NODE_ENV=production npm run blog-posts-block",
-		"starter-page-templates": "calypso-build --source='starter-page-templates'",
-		"dev:starter-page-templates": "npm run starter-page-templates",
-		"build:starter-page-templates": "NODE_ENV=production npm run starter-page-templates",
-		"event-countdown-block": "calypso-build --source='event-countdown-block'",
-		"dev:event-countdown-block": "npm run event-countdown-block",
-		"build:event-countdown-block": "NODE_ENV=production npm run event-countdown-block",
-		"global-styles": "calypso-build --source='global-styles'",
-		"build:global-styles": "NODE_ENV=production npm run global-styles",
-		"dev:global-styles": "npm run global-styles",
-		"jetpack-timeline": "calypso-build --source='jetpack-timeline'",
-		"dev:jetpack-timeline": "npm run jetpack-timeline",
-		"build:jetpack-timeline": "NODE_ENV=production npm run jetpack-timeline",
-		"wpcom-block-editor-nux": "calypso-build --source='wpcom-block-editor-nux'",
-		"dev:wpcom-block-editor-nux": "npm run wpcom-block-editor-nux",
-		"build:wpcom-block-editor-nux": "NODE_ENV=production npm run wpcom-block-editor-nux",
-		"dev": "node bin/npm-run-build.js --dev",
-		"build": "node bin/npm-run-build.js --build",
-		"test:js": "npx wp-scripts test-unit-js --config='bin/js-unit-config.js' --colors",
-		"test:js:help": "npx wp-scripts test-unit-js --config='bin/js-unit-config.js' --help --colors",
-		"test:js:watch": "npx wp-scripts test-unit-js --config='bin/js-unit-config.js' --watch --colors",
-		"test:js:update-snapshots": "npx wp-scripts test-unit-js -u --config='bin/js-unit-config.js' --colors",
-		"test:e2e": "npx wp-scripts test-e2e --wordpress-base-url='http://localhost:4013' --colors",
-		"test:e2e:watch": "npm run test-e2e -- --watch",
-		"clean": "npx rimraf dist full-site-editing-plugin/*/dist",
-		"prebuild": "npm run clean",
-		"predev": "npm run clean",
-		"sync:blog-posts-block": "./bin/sync-blog-posts-block.sh",
-		"wpcom-sync": "./bin/wpcom-watch-and-sync.sh",
-		"prepare": "npm run sync:blog-posts-block --nodemodules"
+		"full-site-editing": "check-npm-client && calypso-build --source='dotcom-fse'",
```

Comment:
All these changes would be so much cleaner if package.json had support for `prerun`... oh well 🤷‍♂